torcharrow/functional.py

```diff
@@ -104,6 +104,25 @@ def __getattr__(op_name: str):
     setattr(sys.modules["torcharrow.functional"], op_name, wrapper)
     return wrapper
 
+### operations in for text domain
+def add_tokens(
+    input_col: Union[ListColumn, List[Union[int, str]]],
+    tokens: Union[ListColumn, List[Union[int, str]]],
+    begin: bool,
+) -> NumericalColumn:
+    """
+    Append or prepend a token to a column. 
```

Comment:
thanks for adding the doc string!

Reply:
QQ: should we add each UDF we implement to the functional alongside documentation? I.e. would it be best practice for us to do this with `bpe_tokenize` and `lookup_indices` UDFs?

Reply:
@wenleix lmk if you have any thoughts on this

Reply:
> should we add each UDF we implement to the functional alongside documentation?

Yes. But since `bpe_tokenize` and `lookup_indices` don't build by default in TorchArrow package, so we can defer adding it a bit.